whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisSearchServiceTestIntegration.java

```diff
@@ -216,6 +214,28 @@ public void search_accept_json() {
         assertThat(whoisObject.getPrimaryKey().get(0).getValue(), is("TP1-TEST"));
     }
 
+    @Test
+    public void search_accept_text_plain() {
```

Comment:
Please add tests with multiple objects in the response (e.g. an inetnum and a role), including comment(s)

Reply:
Also add a test for a lookup of a non-existant object, what's in the response body? Does it include `%ERROR:101: no entries found` ?

Reply:
Thanks for adding tests

Reply:
done, please if there is more required test let me know